BeyFaith/my-brand
# Merge to gh-Pages

### What does this PR do?
This PR will let gh-pages host my website online.

### Description of Task to be completed?
Hosted my website online via gh-pages
### How should this be manually tested?
Go to github/BeyFaith/my-brand ,go to settings and you will see gh-pages click on the link that has been established and you will be directed to the project.

### Any background context you want to provide?
Icons may not render because the Fontawesome folder is in the develop branch. Will be available as soon as they are merged with develop branch.
